### ISSUE TITLE
Broad Glob Patterns content config throws warning when anything containing the word `vendor` is in the specified folder

### ISSUE THREAD

**Issue by SolidAnonDev** [tailwindlabs/tailwindcss#14149]
**What version of Tailwind CSS are you using?**

`latest: 3.4.8`

**What build tool (or framework if it abstracts the build tool) are you using?**

```
"next": "13.5.1",
    "postcss": "^8.4.41",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "tailwindcss": "^3.4.8",
```

**What version of Node.js are you using?**

`>18`

**What browser are you using?**

N/A

**What operating system are you using?**

Windows

**Reproduction URL**

[Reproduction Link](https://stackblitz.com/edit/stackblitz-starters-tvynhn?file=app%2Fvendor.ts)

**Describe your issue**

I am running a project and have a database entity called `vendor`, thus I have a few files containing the word `vendor`, though I have no direct matches specifically. Things like `vendor-service.ts` and `edit-vendor-form.tsx` including a folder named `edit-vendors/` as a file-routed page and so on.

Any folder or file containing the word `vendor` in any of the folders included in the glob pattern in `tailwind.config.js` will result in the following warning thrown:

```
warn - Your `content` configuration includes a pattern which looks like it's accidentally matching all of `vendor` and can cause serious performance issues.
warn - Pattern: `./app/**/*.ts`
warn - See our documentation for recommendations:
warn - https://tailwindcss.com/docs/content-configuration#pattern-recommendations
```

This is related to the warnings included in broad glob patterns in #14140 - pointed out [in this comment](https://github.com/tailwindlabs/tailwindcss/discussions/14145#discussioncomment-10269135) by @adamwathan that vendor is the node_modules equivalent in PHP projects. 

### Instructions to reproduce

The error should show the first time stackblitz project loads and compiles `/page`, but if you miss it or don't see it:

1. Open Stackblitz Link - allow dependencies to install
2. _Stop the server in the terminal using Ctrl+C_
3. Start again using the command `npm run dev`
4. When `/page` compiles, you will see the above warning thrown

If you want to experiment, you will notice you can delete the files and folders containing the word `vendor` and the issue will stop.

**Comment by RobinMalfait** [comment]
Hey!

This should be fixed by https://github.com/tailwindlabs/tailwindcss/pull/14147 and is released as [v3.4.9](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.9).

This issue should go away when you install the latest version 👍

**Comment by SolidAnonDev** [comment]
> Hey!
> 
> This should be fixed by #14147 and is released as [v3.4.9](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.9).
> 
> This issue should go away when you install the latest version 👍


Thank you for all your hard work!
